src/common/hostProcess.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @name - Host Process
+ * @description - Describes the host process
+ * @member type - The type of the host process (electron, browser, etc)
+ * @member release - The release string of the host process
+ */
+export interface IHostProcess {
+    type: HostProcessType;
+    release: string;
+}
+
+/**
+ * @enum ELECTRON - Electron Host Process Type
+ * @enum BROWSER - Browser Host Process Type
+ */
+export enum HostProcessType {
+    Electron = 1, // bits: 01
+    Browser = 2,  // bits: 10
+    All = 3,      // bits: 11
+}
+
+import os from "os";
+const osRelease = os.release().toLowerCase();
+
+function getHostProcessType(): HostProcessType {
```

Comment:
@JordanEllis6809 was this resolved?

Reply:
No there was a backlog item added for it though.

Testing this requires stubbing out the os module twice within one test, which requires using `require` instead of `import`. This was an approach that was previously taken in this PR's commit history, and was something we didn't want to do as it went against our style guide. A good solution that tests what we want to test isn't immediately obvious...